lib/core/device.js

```diff
@@ -217,7 +217,11 @@ device.detectUsb = function(driverId, cb) {
     vendorId: driverManifest.vendorId,
     productId: driverManifest.productId
   };
-  chrome.usb.getDevices(identification, function(results) {
```

Comment:
Just revisited this and it looks like we'll finally be able to get vendor ID and product ID using `chrome.serial.getDevices` when Chrome 48 get released.

Current Chrome 46:
<img width="254" alt="screen shot 2015-11-11 at 15 53 36" src="https://cloud.githubusercontent.com/assets/418233/11096416/c20d050c-8891-11e5-92b3-2b637901a656.png">

Chrome 48 (canary build):
<img width="265" alt="screen shot 2015-11-11 at 15 53 22" src="https://cloud.githubusercontent.com/assets/418233/11096425/cf56a0a6-8891-11e5-9bd9-cbe162a4b29e.png">

This will resolve the `WARNING: More than one device found for ...` issue we've been having.